homeassistant/components/google_assistant/trait.py

```diff
@@ -843,7 +843,7 @@ def query_attributes(self):
         response = {}
         unit = self.hass.config.units.temperature_unit
         current_temp = self.state.state
-        if current_temp not in (STATE_UNKNOWN, STATE_UNAVAILABLE):
```

Comment:
This is odd, it should not be possible to be None. I feel this is more or less working around / masking the real issue. Which integration causes this / provides this sensor?